tests/runtest.sh

```diff
@@ -471,29 +471,31 @@ declare -a playlistTests
 function read_array {
     local theArray=()
 
+    if [ ! -f "$1" ]; then
+        return
+    fi
+
     # bash in Mac OS X doesn't support 'readarray', so using alternate way instead.
     # readarray -t theArray < "$1"
+    # Any line that starts with '#' is ignored.
     while IFS='' read -r line || [ -n "$line" ]; do
-        theArray[${#theArray[@]}]=$line
+        if [[ $line != "#"* ]]; then
```

Comment:
Thank you for this addition! It is unfortunate we are creating another metadata structure for failing tests though.

Reply:
Well, just extending/generalizing the existing structure.

Yes, I'd like to see this go away in favor of some kind of unification...